lib/plugins/helper/date.js

```diff
@@ -2,7 +2,7 @@
 
 const moment = require('moment-timezone');
 const isMoment = moment.isMoment;
-const isDate = require('lodash').isDate;
+const isDate = require('lodash/isDate');
```

Comment:
that's right!
In addition, `lodash.isDate()` returns the result if `util.types.isDate()` exists, if one exists.
Although it may be corrected, I think that the necessity to thoroughly perform the contents of this PR is thinner than other libraries such as `warehouse`.
That's because `hexo` exports the `lodash` namespace and it's a big size as a library.